app/commands/v2/put_content.rb

```diff
@@ -222,11 +220,13 @@ def send_downstream(content_item)
         message += "{ content_store: Adapters::DraftContentStore, content_item_id: #{content_item.id} }"
         logger.info message
```

Comment:
Is it worth moving this log line into the lambda?  I presume they're going to execute pretty quickly but technically it's lying.